helm/azure-operator-resource-chart/values.yaml

```diff
@@ -0,0 +1,14 @@
+clusterName: "test-cluster"
+commonDomain: "godsmack.westeurope.azure.gigantic.io"
+commonDomainResourceGroup: "godsmack"
+hostClusterCIDR: "10.0.0.0/16"
+hostClusterResourceGroup: "godsmack"
+hostClusterVirtualNetwork: "godsmack"
```

Comment:
Might want to remove those installation specific values

Reply:
I think this is only maybe wrong in political sense.